src/js/components/Layer/__tests__/__snapshots__/Layer-test.js.snap

```diff
@@ -900,8 +900,8 @@ exports[`Layer dark context 1`] = `
   -ms-flex-direction: column;
   flex-direction: column;
   min-height: 48px;
-  background-color: #FFFFFF;
-  color: #444444;
+  background-color: #000000;
+  color: #f8f8f8;
```

Comment:
This snapshot change is suspicious, can you please verify it's the correct behavior for the test?

Reply:
I believe that this is the correct behavior. Previously in the test the Box has a dark background and the Layer has a light background with dark text. Now the Box and Layer have a dark background and the text in the Layer is light.

Reply:
<img width="1351" alt="Screen Shot 2020-10-26 at 11 51 02 AM" src="https://user-images.githubusercontent.com/54560994/97209181-a4eabf80-1781-11eb-8292-20603884ca76.png">